src/vdaf.rs

```diff
@@ -168,6 +170,14 @@ pub trait Aggregator: Vdaf {
         agg_param: &Self::AggregationParam,
         output_shares: M,
     ) -> Result<Self::AggregateShare, VdafError>;
+
+    /// Accumulates the sequence of output shares into the accumulator.
+    fn accumulate(
```

Comment:
This is what we were talking through in Slack. I think we were both leaning towards putting some kind of `aggregate` method on trait `Aggregatable` that would add an `OutputShare` to an `AggregateShare`. I want to try implementing the Hits VDAF to see what ergonomics I want in that case.

Reply:
Ack. I think the trait the method would be more natural on `Aggregatable`, but we can leave this open for now. I'm mainly worried about introducing yet another term, `accumulator`. for `output_share`. We could avoid this if instead we extended `Aggregatable`. Something like:
```rust
pub trait Aggregatable {
+    /// NEW
+    type OutputShare;

    /// Update an aggregate share by merging it with another (`agg_share`).
    fn merge(&mut self, agg_share: &Self) -> Result<(), VdafError>;

+    /// NEW
+    fn accumulate(&mut self, output_share: &OutputShare) -> Result<(), VdafError>;   
} 
```